solidity/deploy/41_deploy_multi_asset_vault.ts

```diff
@@ -0,0 +1,46 @@
+import type { DeployFunction } from "hardhat-deploy/types"
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import { waitForTransaction } from "../helpers/deployment"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments, helpers } = hre
+  const { deployer, governance } = await helpers.signers.getNamedSigners()
+
+  const mezoPortal = await deployments.get("MezoPortal")
+
+  const initialSupportedAssets = [
+    "0x7A56E1C57C7475CCf742a1832B028F0456652F97", // SolvBTC
+    "0xd9D920AA40f578ab794426F5C90F6C731D159DEf", // SolvBTC.BBN
+  ]
+
+  const [_, deployment] = await helpers.upgrades.deployProxy(
```

Comment:
Don't we want to check [if it was already deployed](https://github.com/thesis/mezo-portal/blob/24ebdee07b02e89de1db831d774bebc4319fd9bb/solidity/deploy/03_deploy_portal.ts#L12-L15)? Could save some pain with `func.skip` on various networks.

Reply:
4831b57462b9773fcdf8061b1d3617b754d33508